packages/gallery/src/components/item/videos/videoItem.js

```diff
@@ -231,10 +232,12 @@ class VideoItem extends GalleryComponent {
               disablepictureinpicture: 'true',
               muted: !this.props.styleParams.videoSound,
               preload: 'metadata',
-              poster: this.props.createUrl(
-                GALLERY_CONSTS.urlSizes.SCALED,
-                GALLERY_CONSTS.urlTypes.HIGH_RES
-              ),
+              poster: shouldCreateVideoPlaceholder(this.props.styleParams)
+                ? this.props.createUrl(
+                    GALLERY_CONSTS.urlSizes.SCALED,
+                    GALLERY_CONSTS.urlTypes.HIGH_RES
+                  )
+                : '',
```

Comment:
I see this pattern coming up a lot in the code: 

1. There will always be a `poster` attribute 
2. When the `poster` isn't relevant it will have a "no meaning" value such as `''`. 


My suggestion is to create a `PosterVideoItem` that extends `VideoItem` and will always have relevant value. 
The `PosterVideoItem` will be created in an upper level based on `shouldCreateVideoPlaceholder`